docs/user-guide/devguide.md

```diff
@@ -300,6 +301,21 @@ You can even test sending Cloud-2-Device message (e.g. by VSCode right click on
 
 The Arduino example provided above will print the message on the console. Keep in mind that a [LoRaWAN Class A](https://www.thethingsnetwork.orglorawan/) device will only receive after a transmit, in our case every 30 seconds.
 
+### Observability
+
+We support Azure Monitor for observability of the LoRaWAN starter kit. If you decide to use Azure Monitor, you will need to create an Application Insights instance and a Log Analytics workspace in your subscription. To enable observability, modify the following settings in your `.env` file:
+
+```{bash}
+APPINSIGHTS_INSTRUMENTATIONKEY={appinsight_key}
+IOT_HUB_RESOURCE_ID=/subscriptions/{subscription_id}/resourceGroups/{resource_group}/providers/Microsoft.Devices/IotHubs/{iot_hub_name}
+LOG_ANALYTICS_WORKSPACE_ID={log_analytics_workspace_id}
+LOG_ANALYTICS_SHARED_KEY="..."
+```
+
+Generate a deployment manifest from `deployment_observability.layered.template.json` and deploy it to the edge devices for which you want to apply the observability. The template will set up the metrics collector module on the edge and connect it with your Log Analytics instance. The gateway will connect to your Application Insights instance. Please make sure that you set the `APPINSIGHTS_INSTRUMENTATIONKEY` also before deploying the `deployment.template.lbs.json` solution, if you want to make sure that the gateway can connect to Application Insights. The Application Insights log level will always be the same as the console log level.
```

Comment:
nit: add link to metrics collector module docs: https://docs.microsoft.com/en-us/azure/iot-edge/how-to-collect-and-transport-metrics?view=iotedge-2020-11&tabs=iothub#metrics-collector-module